apps/communications/contacts/js/views/list.js

```diff
@@ -808,6 +812,36 @@ contacts.List = (function() {
 
     loadedContacts[contact.id][group] = contact;
 
+    // If the contact is in the cache but nothing changed, we bail out.
+    // Otherwise, we remove the DOM node and let the flow continue so we
+    // can append the new node with the updated information.
+    if (inCache) {
+      var cachedContact = Cache.getContact(contact.id);
+      // We don't add Facebook information while creating the placeholder,
+      // this information is added afterwards in renderFbData.
+      // Because of that, we need to get rid of the additional information
+      // from the cached contact before checking against the original contact
+      // information.
+      if (contact.category.indexOf('facebook') != -1) {
```

Comment:
I don't know, it was the easiest way I saw to check that, but please let me know if there's a better way :)